src/learn/math/drill/keyboard.jsx

```diff
@@ -24,8 +29,9 @@ class Keyboard extends React.Component {
     }, {});
 
     if (props.largeKeyboard) {
```

Comment:
I've created 2 styles to apply here because I was getting an error before.

error: `Cannot add property height, object is not extensible`

So, here I only did a copy of them to buttonStyle.